File: Iniciante/c/problema1006.c
// author: Marcela Prata Silverio

#include <stdio.h>

#define peso1 2
#define peso2 3
#define peso3 5

int main(){

    double nota1, nota2, nota3, media; //declaracao das variaveis double

    scanf("%lf", &nota1); //leitura da primeira nota
    scanf("%lf", &nota2); //leitura da segunda nota
    scanf("%lf", &nota3); //leitura da terceira nota

    media = ((nota1 * peso1) + (nota2 * peso2) + (nota3 * peso3))/(peso1 + peso2 + peso3); //calculo da media

    printf("MEDIA = %0.1lf\n", media); //impressao de dados

    return 0;

} 

File: Iniciante/python/problema1006.py
#Autor: Marcela Prata Silverio
#Turma: INF3A

if __name__ == "__main__":

    a = float(input()) #leitura da primeira nota
    b = float(input()) #leitura da segunda nota
    c = float(input()) #leitura da terceira nota

    media = ((a * 2) + (b * 3) + (c * 5)) / (10) #calculo da media ponderada

    print('MEDIA = %.1f' % media) #impressao da media
    